test/integration/build/test_build.py

```diff
@@ -80,6 +81,272 @@ def test_app_without_rockspec(cartridge_cmd, project_without_dependencies):
+    assert all([rock in files for rock in project.rocks_content])
+
+    # with *-scm-2.rockspec in project root
+    os.remove(project.rockspec_path)
+    version = 'scm-2'
+    rockspec_name = '{}-{}.rockspec'.format(project.name, version)
+
+    with open(os.path.join(project.path, rockspec_name), 'w') as f:
+        f.write('''
+                package = '{}'
+                version = '{}'
+                source  = {{ url = '/dev/null' }}
+                dependencies = {{ 'tarantool' }}
+                build = {{ type = 'none' }}
+            '''.format(project.name, version))
+
+    build_logs = [
+        'Build application in',
+        'Running `cartridge.pre-build`',
+        'Running `tarantoolctl rocks make`',
+        'Application was successfully built',
+    ]
+
+    # without --spec
+    cmd = [
+        cartridge_cmd,
+        "build",
+    ]
+
+    rc, output = run_command_and_get_output(cmd, cwd=project.path)
+    assert rc == 0
+    assert all([log in output for log in build_logs])
+    assert prebuild_output not in output
+
+    # check that all expected rocks was installed
+    files = recursive_listdir(project.path)
+    rocks_content = get_base_project_rocks(project.name, rockspec_name, version)
+    assert '.rocks' in files
+    assert all([rock in files for rock in rocks_content])
+
+    build_logs = [
+        'Build application in',
+        'Running `cartridge.pre-build`',
+        'Running `tarantoolctl rocks make {}`'.format(rockspec_name),
+        'Application was successfully built',
+    ]
+
+    # with --spec
+    cmd = [
+        cartridge_cmd,
+        "build",
+        "--spec",
+        rockspec_name,
+    ]
+
+    rc, output = run_command_and_get_output(cmd, cwd=project.path)
+    assert rc == 0
+    assert all([log in output for log in build_logs])
+    assert prebuild_output not in output
+
+    # check that all expected rocks was installed
+    files = recursive_listdir(project.path)
+    assert '.rocks' in files
+    assert all([rock in files for rock in rocks_content])
+
+
+def test_building_with_two_rockspec_in_project_root(cartridge_cmd, project_without_dependencies):
+    project = project_without_dependencies
+
+    prebuild_output = "pre-build hook output"
+    version = 'scm-2'
+    second_rockspec_name = '{}-{}.rockspec'.format(project.name, version)
+
+    with open(os.path.join(project.path, second_rockspec_name), 'w') as f:
+        f.write('''
+                package = '{}'
+                version = '{}'
+                source  = {{ url = '/dev/null' }}
+                dependencies = {{ 'tarantool' }}
+                build = {{ type = 'none' }}
+            '''.format(project.name, version))
+
+    with open(os.path.join(project.path, 'cartridge.pre-build'), 'w') as f:
+        prebuild_script_lines = [
+            "#!/bin/sh",
+            "echo \"{}\"".format(prebuild_output)
+        ]
+        f.write('\n'.join(prebuild_script_lines))
+
+    build_logs = [
```

Comment:
You shouldn't test all build logs here. Let's concentrate on `tarantoolctl rocks make` in this test. (All other logs are checked in default test, pre-build hook output is used for verbosity flags tests).